zurb-template/src/data/event.json

```diff
@@ -15,18 +15,18 @@
 },
 
 { 
-"text": "Set single or multiple runs for task with count parameter", 
-"link": "https://github.com/intelsdi-x/snap/blob/master/docs/TASKS.md#task-manifest" 
+"text": "2.0 release introduces a new plugin type: streaming collectors", 
+"link": "https://github.com/intelsdi-x/snap/blob/master/docs/STREAMING.md" 
```

Comment:
For now this link does not work. Documentation for streaming is in code review.

Reply:
This link works now that the documentation has been merged.